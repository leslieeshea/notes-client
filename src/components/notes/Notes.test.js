import React from 'react';
import { shallow } from 'enzyme';
import Notes from './Notes';

describe('Notes component', () => {
  it('can render notes', () => {
    const wrapper = shallow(<Notes notes={[{ _id: '123', title: 'hello there', body: 'goodbye' }]}/>);

    expect(wrapper).toMatchSnapshot();
  });
});
