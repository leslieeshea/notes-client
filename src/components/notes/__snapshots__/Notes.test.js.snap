// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notes component can render notes 1`] = `
<ul>
  <li
    key="123"
  >
    <Note
      note={
        Object {
          "_id": "123",
          "body": "goodbye",
          "title": "hello there",
        }
      }
    />
  </li>
</ul>
`;
